hieradata/development.yaml

```diff
@@ -103,11 +102,13 @@ govuk::apps::ckan::priority_worker_processes: '1'
 govuk::apps::ckan::bulk_worker_processes: '1'
 govuk::apps::collections_publisher::enable_procfile_worker: false
 govuk::apps::contacts::extra_aliases: ['contacts-admin']
+govuk::apps::content_data_admin::enabled: true
 govuk::apps::content_store::mongodb_nodes: ['localhost']
 govuk::apps::content_store::mongodb_name: 'content_store_development'
-govuk::apps::content_performance_manager::rabbitmq_user: 'content_performance_manager'
-govuk::apps::content_performance_manager::rabbitmq_hosts: ['localhost']
-govuk::apps::content_performance_manager::rabbitmq::amqp_pass: 'content_performance_manager'
+govuk::apps::content_data_api::rabbitmq_user: 'content_data_api'
+govuk::apps::content_data_api::rabbitmq_hosts: ['localhost']
+govuk::apps::content_data_api::rabbitmq::amqp_pass: 'content_data_api'
+govuk::apps::content_data_api::rabbitmq_password: "%{hiera('govuk::apps::content_data_api::rabbitmq::amqp_pass')}"
```

Comment:
Why do we need this?

Reply:
`govuk::apps::content_data_api::rabbitmq::amqp_pass: 'content_data_api'` sets the rabbitmq password in the development environment, whereas `govuk::apps::content_data_api::rabbitmq_password` is getting the password to be able to access rabbitmq through the app within the development environment.

Reply:
Let's just remove this line - as can't see where its being used

Reply:
Without it, I don't think the Content Data API will know what RabbitMQ password to use: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/apps/content_data_api.pp#L103